backend/app/src/main/kotlin/io/tolgee/api/v2/controllers/V2ProjectsController.kt

```diff
@@ -133,6 +133,7 @@ class V2ProjectsController(
   @GetMapping("/{projectId}/users")
   @Operation(summary = "Returns project all users, who have permission to access project")
   @AccessWithProjectPermission(ProjectPermissionType.MANAGE)
+  @NeedsSuperJwtToken
```

Comment:
True, but I'll keep it since I already wrote a test for this and othervise I would need to create an example page making 2 get requests in the same time.
Github for example asks for password also before you are able to access sensitive pages.